content/2021-08-26-drg-trust/index.md

```diff
@@ -0,0 +1,202 @@
+As you already know and trust this trust anchor you can verify the certificate by using the trust anchor's public key. 
+Once you verified the certificate you can trust it, because the anchor said it's worthy.
+
+That's the jist of it ! X509 have a lot of other details to know about, but that overview is enough for now. 
+
+# Client-side certificates
+
+Drogue-cloud support authenticating devices using client certificates.
+The use of X.509 certificates simplifies the process of device authentication, providing a one-to-many relationship between an application and its devices.
+X.509 certificates provide stronger client authentication compared to access tokens or username-password combinations because the private key never leaves the device.
+
+## Client certificates 
+
+The authentication flow using client-side certificate is the same as explained earlier: the device, when connecting to drogue-cloud present a certificate. 
+Drogue-cloud then verifies the signature on the certificate to authenticate the device.
+
+## Application trust anchor
+
+To be able to verify the certificate we need to set up a common trust anchor between the device and Drogue cloud.
+We will do that by generating a public/private key pair. This key pair is used to sign a certificate also called a self-signed certificate. This certificate is then attached to an application. 
+
+Now, the private key of the application can be used to sign certificates of devices, which will then be presented at the time of authentication.
+
+# Setting things up 
+
+In this example we will use our Drogue command line tool - `drg` as it makes some steps easier. 
+See how to install drg and login to a Drogue Cloud instance [here](https://github.com/drogue-iot/drg#installation).
+To know more about how to use `drg` for managing apps and devices, you can check out this [blog](https://blog.drogue.io/drg-guided-tour/).
+
+**DISCLAIMER**: Some features used in this walkthrough are not yet released. To reproduce them you'll need to install `drg` from source and deploy drogue-cloud the latest availables images.
+You can try to use our [developpment deployment](https://console-drogue-dev.apps.wonderful.iot-playground.org/), at your own risk ! 
+
+We will also show how to set up a trust anchor manually.
+
+You will need a running drogue-cloud instance obviously ! You can use the [sandbox](https://sandbox.drogue.cloud/) we provide.
+
+## Add the application trust anchor
+
+If you don't already have , let's create an application and call it `demo-app`:
+```
+drg create app demo-app
+```
+
+Now, we need to create a keypair and a self-signed certificate to add a new trust anchor (a.k.a. CA certificate) to the application object.
+We also specify a 'key-output' to save the private key on our system. This key file is important as it would be used later to sign device certificates.
```

Comment:
```suggestion
We also specify a 'key-output' to save the private key on our system. This key file is important, as it will be used later to sign device certificates.
```